### ISSUE TITLE
Unable to do v2 releases

### ISSUE THREAD

**Issue by jonkoops** [JedWatson/classnames#393]
@JedWatson could you please enable releases from the `v2` branch? See #364

**Comment by dcousens** [comment]
@jonkoops what if we release from `main` anyway?

**Comment by jonkoops** [comment]
Yeah, we could do that. We'd have to push what we have on `main` now to `v3` branch, and then push the `v2` branch to `main`. We'll still be in trouble if we want to continue to maintain `v2` in the future, but hopefully @JedWatson has given us the keys to the castle by then.

The only problem might be that there could be some branch protection set up for `main` that would prevent this. WDYT?

**Comment by BearCooder** [comment]
So this seems to be basically not maintainable because you dont have the proper access? @jonkoops @JedWatson

**Comment by jonkoops** [comment]
It blocks us from doing a v3 release and maintain a v2 at the same time, yes.

**Comment by BearCooder** [comment]
He is pretty active on his Twitter that is linked in his github. maybe a DM on Twitter would help. I do not have an account there though
